### ISSUE TITLE
MQTT Explorer shows values but homeassistant does not create any sensors / entities

### ISSUE THREAD

**Issue by thetschulian** [z-master42/solarflow#38]
hello,  i cannot get the integration of zendure working. I am using Hub2000.
I already connected my openDTU to the mqtt broker of homeassistant. this works.

now I am stuck on zendure.

I changed the options in the configuration of the core mosquitto
![image](https://github.com/user-attachments/assets/c798f71a-f210-4878-bb5a-b9469c819b2a)

I created the config
![image](https://github.com/user-attachments/assets/0f38a425-8632-44b5-963e-6b23a38d3d24)
![image](https://github.com/user-attachments/assets/11d56bfb-1257-4b48-a081-5ee96a968428)

homeassistant trys to connect (without any error)

2024-11-16 12:54:50: Connecting bridge zendure-broker (mqtt.zen-iot.com:1833)
2024-11-16 13:02:59: Connecting bridge zendure-broker (mqtt.zen-iot.com:1833)

mqtt explorer shows of values

![image](https://github.com/user-attachments/assets/6731e4d6-9232-4461-828c-4e105c775939)

**Comment by z-master42** [comment]
What is the question now?

**Comment by thetschulian** [comment]
the question is, why my sensors are not generated

to your second question: yes, I can 

![image](https://github.com/user-attachments/assets/f4602ca9-5175-47e8-bbae-314a68aab938)

**Comment by z-master42** [comment]
Hmm, I don't see any Zendure related topics.

**Comment by thetschulian** [comment]
I tried everything around and I get no zendure topics in my homeassistant. may u have a tipp? 

I did everything like in your tutorial. ...the zendure.conf is loaded . i can see this in the log of the mqtt broker

**Comment by z-master42** [comment]
Hmm, that's strange. I can't see anything that looks wrong.
The hub must definitely be online for values to be transmitted. And if the connection between the two brokers has been successfully established, the same entries must also appear in Mosquitto.

There must also be several entries from Zendure under the topic homeassistant/sensor and homeassistant/switch.
As well as the topic with ‘Device-ID’/state.

**Comment by thetschulian** [comment]
I reinstalled everything with same result.

Looks like the trys to connect to the zendure broker frequently with no errors.


2024-11-18 20:10:35: Client <unknown> closed its connection.
2024-11-18 20:11:19: Connecting bridge zendure-broker-github-test (mqtt.zen-iot.com:1833)
2024-11-18 20:12:35: New connection from 172.30.32.2:48134 on port 1883.
2024-11-18 20:12:35: Client <unknown> closed its connection.
2024-11-18 20:12:50: Connecting bridge zendure-broker-github-test (mqtt.zen-iot.com:1833)
2024-11-18 20:14:21: Connecting bridge zendure-broker-github-test (mqtt.zen-iot.com:1833)
2024-11-18 20:14:35: New connection from 172.30.32.2:59092 on port 1883.
2024-11-18 20:14:35: Client <unknown> closed its connection.
2024-11-18 20:15:08: Saving in-memory database to /data//mosquitto.db.
2024-11-18 20:15:52: Connecting bridge zendure-broker-github-test (mqtt.zen-iot.com:1833)
2024-11-18 20:16:35: New connection from 172.30.32.2:51510 on port 1883.
2024-11-18 20:16:35: Client <unknown> closed its connection.
2024-11-18 20:17:14: Connecting bridge zendure-broker-github-test (mqtt.zen-iot.com:1833)
2024-11-18 20:18:31: Connecting bridge zendure-broker-github-test (mqtt.zen-iot.com:1833)
2024-11-18 20:18:35: New connection from 172.30.32.2:42116 on port 1883.
2024-11-18 20:18:35: Client <unknown> closed its connection.
2024-11-18 20:19:54: Connecting bridge zendure-broker-github-test (mqtt.zen-iot.com:1833)
2024-11-18 20:20:35: New connection from 172.30.32.2:56820 on port 1883.
2024-11-18 20:20:35: Client <unknown> closed its connection.
2024-11-18 20:21:25: Connecting bridge zendure-broker-github-test (mqtt.zen-iot.com:1833)
2024-11-18 20:22:35: New connection from 172.30.32.2:58802 on port 1883.
2024-11-18 20:22:35: Client <unknown> closed its connection.
2024-11-18 20:22:56: Connecting bridge zendure-broker-github-test (mqtt.zen-iot.com:1833)
2024-11-18 20:24:27: Connecting bridge zendure-broker-github-test (mqtt.zen-iot.com:1833)
2024-11-18 20:24:35: New connection from 172.30.32.2:53610 on port 1883.
2024-11-18 20:24:35: Client <unknown> closed its connection.
2024-11-18 20:25:58: Connecting bridge zendure-broker-github-test (mqtt.zen-iot.com:1833)
2024-11-18 20:26:35: New connection from 172.30.32.2:33052 on port 1883.
2024-11-18 20:26:35: Client <unknown> closed its connection.
2024-11-18 20:27:29: Connecting bridge zendure-broker-github-test (mqtt.zen-iot.com:1833)
2024-11-18 20:28:35: New connection from 172.30.32.2:45466 on port 1883.
2024-11-18 20:28:35: Client <unknown> closed its connection.


I checked my app, I use the global region 

so i used this command

curl -i -v --json "{'snNumber': 'HO1Hchanged', 'account': 'changed@outlook.com'}" https://app.zendure.tech/v2/developer/api/apply

to get 

{"code":200,"success":true,"data":{"appKey":"mykey","secret":"mysecret","mqttUrl":"mqtt.zen-iot.com","port":1883},"msg":"Operation successful"}* Connection #0 to host app.zendure.tech left intact

very strange.


EDIT: beim schreiben und  nochmal durcvhgehen ist mir aufgefallen, dass ich

(mqtt.zen-iot.com:1833)  verwendet habe.. es sollte jedoch 1883 heißen ...
